sdk/storage/storage-file-share/samples/typescript/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "moduleResolution": "node",
+
+    "lib": ["dom", "dom.iterable", "esnext.asynciterable"],
+
+    "allowSyntheticDefaultImports": true,
+
+    "outDir": "dist"
+  },
+  "include": ["**/*.ts"]
```

Comment:
I don't think this is necessary, the default is to spider for all TS files starting with the workspace root, right?